raiden/connection_manager.py

```diff
@@ -164,7 +187,7 @@ def wait_for_settle(self, closed_channels):
 
     def join_channel(self, partner_address, partner_deposit):
         """Will be called, when we were selected as channel partner by another
-        node. It will fund the channel with up to the partner's deposit, but
+        node. It will fund the channel with up to the partners deposit, but
```

Comment:
ehm @palango This was not a typo. It was fine before.

Reply:
Are you sure? Why is this right here?

Reply:
Possessive case in English. A form of the genitive case used in a genitive-noun relationship.

You put the `'s` at the end to indicate posession. In our case to indicate that the deposit belongs to the partner.
There are some exceptions to this rule but they don't apply here.

You could maintain the same semantics by using genitive with composition and turn it into:

> with up to the deposit **of** the partner

tl;dr: Yes I am sure.

Reply:
Good to know.